multipaz-compose/src/commonMain/kotlin/org/multipaz/compose/camera/CameraCaptureResolution.kt

```diff
@@ -0,0 +1,16 @@
+package org.multipaz.compose.camera
+
+/**
+ * Supported [Camera] resolution (frame size) options (as might be needed for different use cases).
+ * The width and height are "desired" value in pixels. Camera hardware may not support certain values, thus the actual
+ * frame size will be selected automatically according to the camera hardware specifications
+ * Default is LOW (640x480), which corresponds to the standard MLKit input image size.
+ *
+ * @param width desired width in pixels.
+ * @param height desired height in pixels.
+ */
+enum class CameraCaptureResolution(val width: Int, val height: Int) {
+    LOW(640, 480),
+    MEDIUM(1280, 720),
+    HIGH(1920, 1080)
+}
```

Comment:
I don't think it's helpful to have `width` or `height` parameters here. Just `LOW`, `MEDIUM`, `HIGH`.

Also missing KDoc for the actual values, e.g.

```
 /** Low resolution, typically 640x480 */
 LOW,
```

Reply:
All I'm saying is that the enum class doesn't need to list the width/height, that can be done in the implementation so it's private/internal, e.g.

```
private CameraCaptureResolution.getDimensions(): Pair<Int, Int> {
   return when(this) {
       LOW -> Pair(640, 480)
       [...]
   }
```